.github/workflows/e2e.yml

```diff
@@ -70,7 +69,8 @@ jobs:
         "$KREW" install krew
         )
         export PATH=$PATH:$HOME/.krew/bin
-        [ -z "$E2E_PARALLELISM" ] && export E2E_PARALLELISM
+        E2E_PARALLELISM=${{ github.event.inputs.parallel }}
+        [ ! -z "$E2E_PARALLELISM" ] && export E2E_PARALLELISM
```

Comment:
When running manually with workflow_dispatch you can set the parallelism you want and it will override the default value. if the current event is not workflow_dispatch or did not precise any parallelism this env var will get ignored and  the e2e tests will be run with the default value

Reply:
How high can the parallelism go in GitHub before we start seeing test failures due to timeout?  I wonder how much value there is in having this configurable.